### PR TITLE
Added SdkType and SdkVersion to header

### PR DESCRIPTION
**Pull request by michaelmaxyz**

### CODE REVIEW

File: lib/lockstep_sdk/lockstep_api.rb [hunk 1]
```diff
@@ -169,6 +169,9 @@ def request(method, path, body, params)
             request["Content-Type"] = 'application/*+json'
             request.body = body
             
+            request["SdkType"] = 'Ruby'
+            request["SdkVersion"] = '2022.3.50.0'
```
Comment (tspence) [inline lib/lockstep_sdk/lockstep_api.rb:181]: As long as we're adding SdkType and SdkVersion, can we please also add MachineName and ApplicationName?

Looks like we can do it via:

```
require 'socket'
hostname = Socket.gethostname
```

https://stackoverflow.com/questions/42566/getting-the-hostname-or-ip-in-ruby-on-rails

**Comment by tspence** [comment]
Awesome!  Have you also submitted a PR to update the code generator pattern?

File: lib/lockstep_sdk/lockstep_api.rb (hunk 1 above)
Comment (michaelmaxyz) [reply lib/lockstep_sdk/lockstep_api.rb:181]: Done!

**Comment by michaelmaxyz** [comment]
> Awesome! Have you also submitted a PR to update the code generator pattern?

I'll do that!

**Comment by michaelmaxyz** [comment]
Just submitted a PR to update code generator!

**Comment by tspence** [comment]
LGTM!

**Pull request merged**